app/views/content/_show.html.erb

```diff
@@ -28,6 +28,22 @@ set_meta_tags title: content.name, description: content.description
             <li class="tab col s3 <%= "disabled" unless data[:attributes].any? { |m| content.send(m).present? } %>"><a href="#<%= category %>"><%= category.to_s.humanize %></a></li>
           <% end %>
         </ul>
+        <%
+          # TODO: Move this to some app-wide initializer
```

Comment:
Would it be better to make an issue card for this? We'd lose track of it like this.